src/mpi/topo/topo_impl.c

```diff
@@ -230,8 +201,8 @@ int MPIR_Cart_get(MPIR_Comm * comm_ptr, int maxdims, int dims[], int periods[],
     goto fn_exit;
 }
 
-int MPIR_Cart_map(const MPIR_Comm * comm_ptr, int ndims, const int dims[],
-                  const int periodic[], int *newrank)
+int MPIR_Cart_map_impl(MPIR_Comm * comm_ptr, int ndims, const int dims[],
+                       const int periods[], int *newrank)
```

Comment:
This change `periods` breaks the code (`MPL_UNREFERENCED_ARG(periodic);`).

Reply:
Should this change be in this commit?

Reply:
> 
> 
> This change `periods` breaks the code (`MPL_UNREFERENCED_ARG(periodic);`).

Both changed to `periods`. I would like `MPIR_Xxx_impl` use the same name as `MPI_Xxx`, for consistency